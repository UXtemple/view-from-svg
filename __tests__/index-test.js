import fromSvg from '../index.js'
import fs from 'fs'

const sources = [`${__dirname}/f.svg`, `${__dirname}/f2.svg`]

describe('fromSvg', () => {
  sources.forEach(file => {
    const raw = fs.readFileSync(file, 'utf-8')

    it(`${file}:\n${raw}`, () => {
      expect(fromSvg(raw)).toMatchSnapshot()
    })
    it(`${file} legacy:\n${raw}`, () => {
      expect(fromSvg(raw, true)).toMatchSnapshot()
    })
  })
})
