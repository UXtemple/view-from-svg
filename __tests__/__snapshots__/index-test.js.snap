exports[`fromSvg /Users/craverod/views/morph/from-svg/__tests__/f.svg legacy:
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 17.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="472px" height="392px" viewBox="0 0 472 392" enable-background="new 0 0 472 392" xml:space="preserve">
<g id="Layer_3">
	<path fill="#FFB564" d="M288.7,187.7c-53.7-32.6-119.8,1-119.8,1s11.6,92.6,11.4,121.4c-0.8,2-1,4.1-0.4,6.3
		c-0.2,1.2-0.6,1.7-1,1.7h1.7c1.1,3,2.9,4.3,5.8,5.3c10.9,4,21.5,6.8,33.1,7.7c3.6,0.3,7.1,0.2,10.5-0.2c1.6,1.5,4,2.4,7.1,2.1
		c13.2-1.2,28-1.9,38.6-10.3c1.7-1.4,2.7-2.7,3.1-4.7h0.8L288.7,187.7z"/>
	<polygon fill="#52E2D7" points="173.2,146.4 180.5,184 206.6,177.2 200.1,144 	"/>
	<path fill="#FFFBE8" d="M222.6,133.8c0,0-17.9-15.6-5.3,39.9c0.6,2.5,19.4,3.4,19.4,3.4L222.6,133.8z"/>
	<path fill="#F9A035" d="M188.6,181.4c0,0-20.6,0-20.6,12.4c0.1,17.2,13.4,105.6,13.1,119.9c-0.2,9.3,19.1,13.6,19.1,13.6"/>
	<path fill="#FFB564" d="M200.2,93.3c0,0,9.7,50.2,11.6,48.4c2-1.8,10.8-7.9,10.8-7.9l-13.5-40.5H200.2z"/>
	<path fill="#FFFBE8" d="M184.1,123.6c0,0-12.9,24.6-10.9,22.8c2-1.8,27-2.4,27-2.4L184.1,123.6z"/>
	<path fill="#F25F68" d="M257.9,92.8c-1.5,2-3,4.1-4.4,6.3c-1.2,1.9-2.9,2.3-4.6,1.9c-1.7,3.4-3.5,6.7-5.7,9.8
		c-6.2,22.4-0.7,41.7-8.7,63.8c-1.3,3.5,0.3,0.3,2.2,2.5c5.6-3,31.1,3.5,38.7,4.2c1.7-2.3-9.1-3.3-10.3-6.6c1.4,3.9,0.2-5.2,0.2-6.1
		c0.3-2.6,0-5.5,0.3-8.1c0.7-4.6-1.4-6.4-0.8-11c1.9-14.5,3.4-29.4,3-44.1c-2.2-3.9-4.1-7.9-5.6-12.1
		C261,92.9,259.4,92.7,257.9,92.8z"/>
</g>
<g id="Layer_2">
	
		<polyline fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="square" stroke-linejoin="round" stroke-miterlimit="10" points="
		238,171.1 245.2,109 260.4,86.9 267.7,110.8 265.1,174.8 	"/>
	
		<polyline fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="square" stroke-linejoin="round" stroke-miterlimit="10" points="
		180.2,182.4 173.8,144 184.1,123.6 200.1,139.2 206.6,176.2 	"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="square" stroke-linejoin="round" stroke-miterlimit="10" d="
		M218.2,174.8c0,0-4.5-16.5-9-36.4c-4.8-21.3-9.7-43.8-9-45.1c1.3-2.6,5.4-4.8,8.9,0c2,2.7,7.9,21.6,13.5,40.5
		c6.3,21.1,12.2,42.3,11.9,40.8"/>
	<path fill="#AF3A46" stroke="#AF3A46" stroke-width="2.4882" stroke-miterlimit="10" d="M200.2,93.3c-11.6-12.2-10.4-19.2-6.6-23
		c6.6-6.6-2.8-16.6-0.9-16.8c6.6-0.6,28.6,9.6,17.3,36.9"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M245.2,109c0,0,11.3-4.8,22.5,1.9"/>
	
		<path fill="#FFFBE8" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M200.1,139.2c0,4.7-12.2,4.7-12.2,4.7c-3.2,5.3-14.1,0-14.1,0"/>
	<path fill="#AF3A46" d="M190.9,130.3c3.4,3.3-12,3.5-12,3.5l5.1-10.2L190.9,130.3z"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M211.8,141.6c7.7,0,11.4-6,11.4-6"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M169,188.8L169,188.8c43.9-21.5,94.3-16.6,119.8-1.1L277,321.9c0,0-39.9,23.2-95.2-1.1L169,188.8z"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="4.8228" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M255.7,270.4l-2.2-32.6l-24.4-18.9L205,236.6c0.3,11.8,1.4,22.8,1.7,34.6C206.6,271.2,236,275.9,255.7,270.4z"/>
	
		<line fill="none" stroke="#AF3A46" stroke-width="5" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" x1="229.5" y1="219" x2="229.5" y2="246"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="3.8582" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M233.2,240.8c-1.5,0-2.9,0.3-6.8,0.4c0,0-1-0.9-0.1,5.9c2.9,0.1,4,0.2,6.9,0.2C233.2,247.4,233.2,242.5,233.2,240.8z"/>
</g>
</svg>
 1`] = `
"- block: Svg
  width: 472
  height: 392
  viewBox: 0 0 472 392
  blocks:
  - block: Path
    fill: #FFB564
    d: M288.7 187.7c-53.7-32.6-119.8 1-119.8 1s11.6 92.6 11.4 121.4c-.8 2-1 4.1-.4 6.3-.2 1.2-.6 1.7-1 1.7h1.7c1.1 3 2.9 4.3 5.8 5.3 10.9 4 21.5 6.8 33.1 7.7 3.6.3 7.1.2 10.5-.2 1.6 1.5 4 2.4 7.1 2.1 13.2-1.2 28-1.9 38.6-10.3 1.7-1.4 2.7-2.7 3.1-4.7h.8l9.1-130.3z

  - block: Path
    fill: #52E2D7
    d: M173.2 146.4l7.3 37.6 26.1-6.8-6.5-33.2z

  - block: Path
    fill: #FFFBE8
    d: M222.6 133.8s-17.9-15.6-5.3 39.9c.6 2.5 19.4 3.4 19.4 3.4l-14.1-43.3z

  - block: Path
    fill: #F9A035
    d: M188.6 181.4s-20.6 0-20.6 12.4c.1 17.2 13.4 105.6 13.1 119.9-.2 9.3 19.1 13.6 19.1 13.6

  - block: Path
    fill: #FFB564
    d: M200.2 93.3s9.7 50.2 11.6 48.4c2-1.8 10.8-7.9 10.8-7.9l-13.5-40.5h-8.9z

  - block: Path
    fill: #FFFBE8
    d: M184.1 123.6s-12.9 24.6-10.9 22.8c2-1.8 27-2.4 27-2.4l-16.1-20.4z

  - block: Path
    fill: #F25F68
    d: M257.9 92.8c-1.5 2-3 4.1-4.4 6.3-1.2 1.9-2.9 2.3-4.6 1.9-1.7 3.4-3.5 6.7-5.7 9.8-6.2 22.4-.7 41.7-8.7 63.8-1.3 3.5.3.3 2.2 2.5 5.6-3 31.1 3.5 38.7 4.2 1.7-2.3-9.1-3.3-10.3-6.6 1.4 3.9.2-5.2.2-6.1.3-2.6 0-5.5.3-8.1.7-4.6-1.4-6.4-.8-11 1.9-14.5 3.4-29.4 3-44.1-2.2-3.9-4.1-7.9-5.6-12.1-1.2-.4-2.8-.6-4.3-.5z

  - block: G
    blocks:
    - block: Path
      fill: none
      stroke: #AF3A46
      strokeWidth: 4.976
      strokeLinecap: square
      strokeLinejoin: round
      strokeMiterlimit: 10
      d: M238 171.1l7.2-62.1 15.2-22.1 7.3 23.9-2.6 64M180.2 182.4l-6.4-38.4 10.3-20.4 16 15.6 6.5 37M218.2 174.8s-4.5-16.5-9-36.4c-4.8-21.3-9.7-43.8-9-45.1 1.3-2.6 5.4-4.8 8.9 0 2 2.7 7.9 21.6 13.5 40.5 6.3 21.1 12.2 42.3 11.9 40.8
  
    - block: Path
      fill: #AF3A46
      stroke: #AF3A46
      strokeWidth: 2.488
      strokeMiterlimit: 10
      d: M200.2 93.3c-11.6-12.2-10.4-19.2-6.6-23 6.6-6.6-2.8-16.6-.9-16.8 6.6-.6 28.6 9.6 17.3 36.9
  
    - block: Path
      fill: none
      stroke: #AF3A46
      strokeWidth: 4.976
      strokeLinecap: round
      strokeLinejoin: round
      strokeMiterlimit: 10
      d: M245.2 109s11.3-4.8 22.5 1.9
  
    - block: Path
      fill: #FFFBE8
      stroke: #AF3A46
      strokeWidth: 4.976
      strokeLinecap: round
      strokeLinejoin: round
      strokeMiterlimit: 10
      d: M200.1 139.2c0 4.7-12.2 4.7-12.2 4.7-3.2 5.3-14.1 0-14.1 0
  
    - block: Path
      fill: #AF3A46
      d: M190.9 130.3c3.4 3.3-12 3.5-12 3.5l5.1-10.2 6.9 6.7z
  
    - block: Path
      fill: none
      stroke: #AF3A46
      strokeWidth: 4.976
      strokeLinecap: round
      strokeLinejoin: round
      strokeMiterlimit: 10
      d: M211.8 141.6c7.7 0 11.4-6 11.4-6M169 188.8c43.9-21.5 94.3-16.6 119.8-1.1L277 321.9s-39.9 23.2-95.2-1.1l-12.8-132z
  
    - block: Path
      fill: none
      stroke: #AF3A46
      strokeWidth: 4.823
      strokeLinecap: round
      strokeLinejoin: round
      strokeMiterlimit: 10
      d: M255.7 270.4l-2.2-32.6-24.4-18.9-24.1 17.7c.3 11.8 1.4 22.8 1.7 34.6-.1 0 29.3 4.7 49-.8z
  
    - block: Path
      fill: none
      stroke: #AF3A46
      strokeWidth: 5
      strokeLinecap: round
      strokeLinejoin: round
      strokeMiterlimit: 10
      d: M229.5 219v27
  
    - block: Path
      fill: none
      stroke: #AF3A46
      strokeWidth: 3.858
      strokeLinecap: round
      strokeLinejoin: round
      strokeMiterlimit: 10
      d: M233.2 240.8c-1.5 0-2.9.3-6.8.4 0 0-1-.9-.1 5.9 2.9.1 4 .2 6.9.2v-6.5z
  
"
`;

exports[`fromSvg /Users/craverod/views/morph/from-svg/__tests__/f.svg:
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 17.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="472px" height="392px" viewBox="0 0 472 392" enable-background="new 0 0 472 392" xml:space="preserve">
<g id="Layer_3">
	<path fill="#FFB564" d="M288.7,187.7c-53.7-32.6-119.8,1-119.8,1s11.6,92.6,11.4,121.4c-0.8,2-1,4.1-0.4,6.3
		c-0.2,1.2-0.6,1.7-1,1.7h1.7c1.1,3,2.9,4.3,5.8,5.3c10.9,4,21.5,6.8,33.1,7.7c3.6,0.3,7.1,0.2,10.5-0.2c1.6,1.5,4,2.4,7.1,2.1
		c13.2-1.2,28-1.9,38.6-10.3c1.7-1.4,2.7-2.7,3.1-4.7h0.8L288.7,187.7z"/>
	<polygon fill="#52E2D7" points="173.2,146.4 180.5,184 206.6,177.2 200.1,144 	"/>
	<path fill="#FFFBE8" d="M222.6,133.8c0,0-17.9-15.6-5.3,39.9c0.6,2.5,19.4,3.4,19.4,3.4L222.6,133.8z"/>
	<path fill="#F9A035" d="M188.6,181.4c0,0-20.6,0-20.6,12.4c0.1,17.2,13.4,105.6,13.1,119.9c-0.2,9.3,19.1,13.6,19.1,13.6"/>
	<path fill="#FFB564" d="M200.2,93.3c0,0,9.7,50.2,11.6,48.4c2-1.8,10.8-7.9,10.8-7.9l-13.5-40.5H200.2z"/>
	<path fill="#FFFBE8" d="M184.1,123.6c0,0-12.9,24.6-10.9,22.8c2-1.8,27-2.4,27-2.4L184.1,123.6z"/>
	<path fill="#F25F68" d="M257.9,92.8c-1.5,2-3,4.1-4.4,6.3c-1.2,1.9-2.9,2.3-4.6,1.9c-1.7,3.4-3.5,6.7-5.7,9.8
		c-6.2,22.4-0.7,41.7-8.7,63.8c-1.3,3.5,0.3,0.3,2.2,2.5c5.6-3,31.1,3.5,38.7,4.2c1.7-2.3-9.1-3.3-10.3-6.6c1.4,3.9,0.2-5.2,0.2-6.1
		c0.3-2.6,0-5.5,0.3-8.1c0.7-4.6-1.4-6.4-0.8-11c1.9-14.5,3.4-29.4,3-44.1c-2.2-3.9-4.1-7.9-5.6-12.1
		C261,92.9,259.4,92.7,257.9,92.8z"/>
</g>
<g id="Layer_2">
	
		<polyline fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="square" stroke-linejoin="round" stroke-miterlimit="10" points="
		238,171.1 245.2,109 260.4,86.9 267.7,110.8 265.1,174.8 	"/>
	
		<polyline fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="square" stroke-linejoin="round" stroke-miterlimit="10" points="
		180.2,182.4 173.8,144 184.1,123.6 200.1,139.2 206.6,176.2 	"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="square" stroke-linejoin="round" stroke-miterlimit="10" d="
		M218.2,174.8c0,0-4.5-16.5-9-36.4c-4.8-21.3-9.7-43.8-9-45.1c1.3-2.6,5.4-4.8,8.9,0c2,2.7,7.9,21.6,13.5,40.5
		c6.3,21.1,12.2,42.3,11.9,40.8"/>
	<path fill="#AF3A46" stroke="#AF3A46" stroke-width="2.4882" stroke-miterlimit="10" d="M200.2,93.3c-11.6-12.2-10.4-19.2-6.6-23
		c6.6-6.6-2.8-16.6-0.9-16.8c6.6-0.6,28.6,9.6,17.3,36.9"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M245.2,109c0,0,11.3-4.8,22.5,1.9"/>
	
		<path fill="#FFFBE8" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M200.1,139.2c0,4.7-12.2,4.7-12.2,4.7c-3.2,5.3-14.1,0-14.1,0"/>
	<path fill="#AF3A46" d="M190.9,130.3c3.4,3.3-12,3.5-12,3.5l5.1-10.2L190.9,130.3z"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M211.8,141.6c7.7,0,11.4-6,11.4-6"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="4.9763" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M169,188.8L169,188.8c43.9-21.5,94.3-16.6,119.8-1.1L277,321.9c0,0-39.9,23.2-95.2-1.1L169,188.8z"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="4.8228" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M255.7,270.4l-2.2-32.6l-24.4-18.9L205,236.6c0.3,11.8,1.4,22.8,1.7,34.6C206.6,271.2,236,275.9,255.7,270.4z"/>
	
		<line fill="none" stroke="#AF3A46" stroke-width="5" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" x1="229.5" y1="219" x2="229.5" y2="246"/>
	
		<path fill="none" stroke="#AF3A46" stroke-width="3.8582" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" d="
		M233.2,240.8c-1.5,0-2.9,0.3-6.8,0.4c0,0-1-0.9-0.1,5.9c2.9,0.1,4,0.2,6.9,0.2C233.2,247.4,233.2,242.5,233.2,240.8z"/>
</g>
</svg>
 1`] = `
"Svg
width 472
height 392
viewBox 0 0 472 392
Path
fill #FFB564
d M288.7 187.7c-53.7-32.6-119.8 1-119.8 1s11.6 92.6 11.4 121.4c-.8 2-1 4.1-.4 6.3-.2 1.2-.6 1.7-1 1.7h1.7c1.1 3 2.9 4.3 5.8 5.3 10.9 4 21.5 6.8 33.1 7.7 3.6.3 7.1.2 10.5-.2 1.6 1.5 4 2.4 7.1 2.1 13.2-1.2 28-1.9 38.6-10.3 1.7-1.4 2.7-2.7 3.1-4.7h.8l9.1-130.3z

Path
fill #52E2D7
d M173.2 146.4l7.3 37.6 26.1-6.8-6.5-33.2z

Path
fill #FFFBE8
d M222.6 133.8s-17.9-15.6-5.3 39.9c.6 2.5 19.4 3.4 19.4 3.4l-14.1-43.3z

Path
fill #F9A035
d M188.6 181.4s-20.6 0-20.6 12.4c.1 17.2 13.4 105.6 13.1 119.9-.2 9.3 19.1 13.6 19.1 13.6

Path
fill #FFB564
d M200.2 93.3s9.7 50.2 11.6 48.4c2-1.8 10.8-7.9 10.8-7.9l-13.5-40.5h-8.9z

Path
fill #FFFBE8
d M184.1 123.6s-12.9 24.6-10.9 22.8c2-1.8 27-2.4 27-2.4l-16.1-20.4z

Path
fill #F25F68
d M257.9 92.8c-1.5 2-3 4.1-4.4 6.3-1.2 1.9-2.9 2.3-4.6 1.9-1.7 3.4-3.5 6.7-5.7 9.8-6.2 22.4-.7 41.7-8.7 63.8-1.3 3.5.3.3 2.2 2.5 5.6-3 31.1 3.5 38.7 4.2 1.7-2.3-9.1-3.3-10.3-6.6 1.4 3.9.2-5.2.2-6.1.3-2.6 0-5.5.3-8.1.7-4.6-1.4-6.4-.8-11 1.9-14.5 3.4-29.4 3-44.1-2.2-3.9-4.1-7.9-5.6-12.1-1.2-.4-2.8-.6-4.3-.5z

G
Path
fill none
stroke #AF3A46
strokeWidth 4.976
strokeLinecap square
strokeLinejoin round
strokeMiterlimit 10
d M238 171.1l7.2-62.1 15.2-22.1 7.3 23.9-2.6 64M180.2 182.4l-6.4-38.4 10.3-20.4 16 15.6 6.5 37M218.2 174.8s-4.5-16.5-9-36.4c-4.8-21.3-9.7-43.8-9-45.1 1.3-2.6 5.4-4.8 8.9 0 2 2.7 7.9 21.6 13.5 40.5 6.3 21.1 12.2 42.3 11.9 40.8

Path
fill #AF3A46
stroke #AF3A46
strokeWidth 2.488
strokeMiterlimit 10
d M200.2 93.3c-11.6-12.2-10.4-19.2-6.6-23 6.6-6.6-2.8-16.6-.9-16.8 6.6-.6 28.6 9.6 17.3 36.9

Path
fill none
stroke #AF3A46
strokeWidth 4.976
strokeLinecap round
strokeLinejoin round
strokeMiterlimit 10
d M245.2 109s11.3-4.8 22.5 1.9

Path
fill #FFFBE8
stroke #AF3A46
strokeWidth 4.976
strokeLinecap round
strokeLinejoin round
strokeMiterlimit 10
d M200.1 139.2c0 4.7-12.2 4.7-12.2 4.7-3.2 5.3-14.1 0-14.1 0

Path
fill #AF3A46
d M190.9 130.3c3.4 3.3-12 3.5-12 3.5l5.1-10.2 6.9 6.7z

Path
fill none
stroke #AF3A46
strokeWidth 4.976
strokeLinecap round
strokeLinejoin round
strokeMiterlimit 10
d M211.8 141.6c7.7 0 11.4-6 11.4-6M169 188.8c43.9-21.5 94.3-16.6 119.8-1.1L277 321.9s-39.9 23.2-95.2-1.1l-12.8-132z

Path
fill none
stroke #AF3A46
strokeWidth 4.823
strokeLinecap round
strokeLinejoin round
strokeMiterlimit 10
d M255.7 270.4l-2.2-32.6-24.4-18.9-24.1 17.7c.3 11.8 1.4 22.8 1.7 34.6-.1 0 29.3 4.7 49-.8z

Path
fill none
stroke #AF3A46
strokeWidth 5
strokeLinecap round
strokeLinejoin round
strokeMiterlimit 10
d M229.5 219v27

Path
fill none
stroke #AF3A46
strokeWidth 3.858
strokeLinecap round
strokeLinejoin round
strokeMiterlimit 10
d M233.2 240.8c-1.5 0-2.9.3-6.8.4 0 0-1-.9-.1 5.9 2.9.1 4 .2 6.9.2v-6.5z

"
`;

exports[`fromSvg /Users/craverod/views/morph/from-svg/__tests__/f2.svg legacy:
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg width="23px" height="27px" viewBox="0 0 23 27" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: sketchtool 41.2 (35397) - http://www.bohemiancoding.com/sketch -->
    <title>77022E8A-9E68-4FC1-BD95-03DCA3BE4C19</title>
    <desc>Created with sketchtool.</desc>
    <defs></defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="RCE-Icons-" transform="translate(-685.000000, -418.000000)">
            <g id="Edit-notes" transform="translate(685.000000, 418.000000)">
                <g id="Group-Copy">
                    <g id="Group-7">
                        <polygon id="Shape" fill="#565757" points="2 20.862069 2 5.73959572 9.34206897 5.73959572 9.34206897 3.73959572 0 3.73959572 0 22.862069 19.3793103 22.862069 19.3793103 13.6432818 17.3793103 13.6432818 17.3793103 20.862069"></polygon>
                        <polygon id="Shape" fill="#565757" points="14.8967972 4.76704766 18.3726592 8.1915423 19.7763063 6.76684057 16.3004442 3.34234592"></polygon>
                        <path d="M8.25655172,14.6979786 L8.25655172,12.6920649 L17.6841379,3.4038026 L19.7351404,5.42449465 L10.3225542,14.6979786 L8.25655172,14.6979786 Z M6.25655172,16.6979786 L11.1422734,16.6979786 L22.5848596,5.42449465 L17.6841379,0.596197397 L6.25655172,11.854903 L6.25655172,16.6979786 Z" id="Shape" fill="#565757"></path>
                        <g id="Group-Copy-2" transform="translate(4.360000, 16.896552)" fill="#7F8282">
                            <polygon id="Shape" points="16.3793103 7.27586207 0 7.27586207 0 9.27586207 18.3793103 9.27586207 18.3793103 0.0570749108 16.3793103 0.0570749108"></polygon>
                        </g>
                    </g>
                </g>
            </g>
        </g>
    </g>
</svg>
 1`] = `
"- block: Svg
  width: 23
  height: 27
  viewBox: 0 0 23 27
  blocks:
  - block: G
    fill: none
    fillRule: evenodd
    blocks:
    - block: Path
      fill: #565757
      d: M2 20.862V5.74h7.342v-2H0v19.122h19.38v-9.219h-2v7.22zM14.897 4.767l3.476 3.425 1.403-1.425L16.3 3.342z
  
    - block: Path
      d: M8.257 14.698v-2.006l9.427-9.288 2.051 2.02-9.412 9.274H8.257zm-2 2h4.885L22.585 5.424l-4.9-4.828L6.256 11.855v4.843z
      fill: #565757
  
    - block: Path
      d: M20.74 24.172H4.36v2h18.38v-9.218h-2z
      fill: #7F8282
  
"
`;

exports[`fromSvg /Users/craverod/views/morph/from-svg/__tests__/f2.svg:
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg width="23px" height="27px" viewBox="0 0 23 27" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: sketchtool 41.2 (35397) - http://www.bohemiancoding.com/sketch -->
    <title>77022E8A-9E68-4FC1-BD95-03DCA3BE4C19</title>
    <desc>Created with sketchtool.</desc>
    <defs></defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="RCE-Icons-" transform="translate(-685.000000, -418.000000)">
            <g id="Edit-notes" transform="translate(685.000000, 418.000000)">
                <g id="Group-Copy">
                    <g id="Group-7">
                        <polygon id="Shape" fill="#565757" points="2 20.862069 2 5.73959572 9.34206897 5.73959572 9.34206897 3.73959572 0 3.73959572 0 22.862069 19.3793103 22.862069 19.3793103 13.6432818 17.3793103 13.6432818 17.3793103 20.862069"></polygon>
                        <polygon id="Shape" fill="#565757" points="14.8967972 4.76704766 18.3726592 8.1915423 19.7763063 6.76684057 16.3004442 3.34234592"></polygon>
                        <path d="M8.25655172,14.6979786 L8.25655172,12.6920649 L17.6841379,3.4038026 L19.7351404,5.42449465 L10.3225542,14.6979786 L8.25655172,14.6979786 Z M6.25655172,16.6979786 L11.1422734,16.6979786 L22.5848596,5.42449465 L17.6841379,0.596197397 L6.25655172,11.854903 L6.25655172,16.6979786 Z" id="Shape" fill="#565757"></path>
                        <g id="Group-Copy-2" transform="translate(4.360000, 16.896552)" fill="#7F8282">
                            <polygon id="Shape" points="16.3793103 7.27586207 0 7.27586207 0 9.27586207 18.3793103 9.27586207 18.3793103 0.0570749108 16.3793103 0.0570749108"></polygon>
                        </g>
                    </g>
                </g>
            </g>
        </g>
    </g>
</svg>
 1`] = `
"Svg
width 23
height 27
viewBox 0 0 23 27
G
fill none
fillRule evenodd
Path
fill #565757
d M2 20.862V5.74h7.342v-2H0v19.122h19.38v-9.219h-2v7.22zM14.897 4.767l3.476 3.425 1.403-1.425L16.3 3.342z

Path
d M8.257 14.698v-2.006l9.427-9.288 2.051 2.02-9.412 9.274H8.257zm-2 2h4.885L22.585 5.424l-4.9-4.828L6.256 11.855v4.843z
fill #565757

Path
d M20.74 24.172H4.36v2h18.38v-9.218h-2z
fill #7F8282

"
`;
